rubin_sim/maf/batches/ddfBatch.py

```diff
@@ -130,9 +125,7 @@ def ddfBatch(
             zStep=0.03,
             daymaxStep=3,
             coadd_night=True,
-            gammaName=gammaName,
-            fieldType=fieldType,
-            DD_list=[ddf_sn_fieldnames],
+            gammaName="gamma_DDF.hdf5",
```

Comment:
Seems more appropriate to use the WFD gamma file for the WFD field though?

Reply:
Also - is a sqlconstraint missing here to exclude WFD visits from the bundle? 
(it does make more sense to me to use the WFD visits as well, as it helps fill in epochs, etc., but Philippe set it up this way purposefully and if we want to continue to have consistent results, we'd need to exclude those as well).

Reply:
The point of picking a random spot in the WFD and treating it like a DDF was to sanity check all the DDF metrics. Using the WFD gamma file we got the result that COSMOS would get 13 SNe and a random spot in the WFD would have 80. 

Sure, we can exclude non-ddf observations.